docs/content/docs/custom-resource/overview.md

```diff
@@ -149,6 +149,26 @@ For Session clusters the operator only provides very basic management and monito
  - Monitor overall cluster health
  - Stop / Delete Session cluster
 
+### Cluster Deployment Modes
```

Comment:
This reads ok, however as a consumer of this documentation I might want to know when to pick Native vs Standalone or vice versa. Can we add some basic use-case examples where each mode is useful for?